common/app/views/support/HtmlCleaner.scala

```diff
@@ -697,12 +699,21 @@ case class AtomsCleaner(
             atomContainer.attr("data-atom-id", atomId)
             atomContainer.attr("data-atom-type", atomType)
 
-            val html = if(atomData.isInstanceOf[AudioAtom] && isAdFree(request)) {
-              views.html.fragments.atoms.atom(atomData, Atoms.articleConfig, shouldFence, amp, mediaWrapper, posterImageOverride).toString().replaceAll("flex.acast.com/", "")
+            val articleConfig: ArticleConfiguration = if (atomData.isInstanceOf[AudioAtom]) {
```

Comment:
Hhm, ideally we build one article configuration valid for all atoms.

Reply:
I think we've addressed this now. Will mark as resolved.